app/src/main/java/ch/epfl/skysync/components/ConfirmFlightUI.kt

```diff
@@ -41,7 +44,11 @@ import java.time.format.DateTimeFormatter
  * @param plannedFlight The planned flight for which the confirmation screen is displayed.
  */
 @Composable
-fun confirmationScreen(plannedFlight: PlannedFlight) {
+fun confirmationScreen(
```

Comment:
by convention so far we've used -Screen as a suffix only for the toplevel composables.

Reply:
fixed